libs/qCC_io/FileIOFilter.h

```diff
@@ -293,6 +296,65 @@ class FileIOFilter
 	//! Called when the filter is unregistered
 	/** Does nothing by default **/
 	virtual void unregister() {}
+	
+public:
+	enum FilterFeature
+	{
+		NoFeatures = 0x0000,
+		
+		Import = 0x00001,	//< Imports data
+		Export = 0x0002,	//< Exports data
+		
+		FromPlugin = 0x0004,	//< Implemented in a plugin
+		
+		DynamicInfo = 0x0008,	//< FilterInfo cannot be set statically (this is used for internal consistency checking)
+	};
+	Q_DECLARE_FLAGS( FilterFeatures, FilterFeature )
+	
+protected:
+	struct FilterInfo
+	{
+		//! ID used to uniquely identify the filter (not user-visible)
+		QString mID;
```

Comment:
We don't have this 'm' convention in CloudCompare for members, so it would be better to not mix conventions ;)